maintainers/maintainer-list.nix

```diff
@@ -686,6 +686,16 @@
     githubId = 1078530;
     name = "Alexandre Peyroux";
   };
+  applePrincess = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add applePrincess``.